src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+
+import duke.task.Task;
+
+import duke.exception.MissingDescriptionException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
+public class Duke {
+    private static final TaskManager taskManager = new TaskManager();
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+
+        handleUserInput();
+
+        printGoodbyeMessage();
+    }
+
+    public static void printWelcomeMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Botty McBotface, at your service.");
+        System.out.println("Your wish is my command.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void handleUserInput() {
+        String userInput;
+        String[] processedInput;
+        Scanner in = new Scanner(System.in);
+
+        ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
You may want to consider using plural for the name of that variable since it is an ArrayList of many things.